source/operators/continueWith.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/rxjs-etc
+ */
+
+import { from, Observable, ObservableInput, Subscription } from "rxjs";
+
+const NO_VAL: any = {};
+
+export const continueWith = <T, O>(
+  mapper: (value: T) => ObservableInput<O>
+) => (source$: Observable<T>) =>
```

Comment:
nitpick: Anytime an API involves a function that returns an `ObservableInput`, it's worth using the `ObservedValueOf` mechanism that's in the core. See `concatMap`:

https://github.com/ReactiveX/rxjs/blob/df51b04d7ec68a72b3a4b0d69c3bb29264c72611/src/internal/operators/concatMap.ts#L6-L8

If this is done, functions that return multiple `ObservableInput` types effect an `Observable` with a union element type, like this:

https://github.com/ReactiveX/rxjs/blob/df51b04d7ec68a72b3a4b0d69c3bb29264c72611/spec-dtslint/operators/concatMap-spec.ts#L32-L34

Reply:
Nice! Thanks for letting me know that! I just updated the PR, please let me know if there is anything else that could be improved. Thanks!